plugins/slipping-away-users/frontend/public/javascripts/countly.views.js

```diff
@@ -1,296 +1,154 @@
-                else if (_.isNumber(val) && (val + "").length === 10) {
-                    el.find(".filter-value.date").find("input").val(countlyCommon.formatDate(moment(val * 1000), "DD MMMM, YYYY"));
-                    el.find(".filter-value.date").find("input").data("timestamp", val);
-                }
-                else {
-                    el.find(".filter-value").not(".hidden").find("input").val(val);
-                }
-
-                if (subs[inputs[cur]].length === sub + 1) {
-                    cur++;
-                    sub = 0;
-                }
-                else {
-                    sub++;
-                }
-                total++;
-                if (inputs[cur]) {
-                    $("#filter-add-container").trigger("click");
-                    if (sub > 0) {
-                        setTimeout(function() {
-                            var elChild = $(".query:nth-child(" + (total) + ")");
-                            elChild.find(".and-or").find(".select-items .item[data-value='OR']").trigger("click");
-                            self.setInput(inputs, subs, cur, sub, total);
-                        }, 500);
-                    }
-                    else {
-                        self.setInput(inputs, subs, cur, sub, total);
-                    }
-                }
-                else {
-                    setTimeout(function() {
-                        $("#apply-filter").removeClass("disabled");
-                        $("#no-filter").hide();
-                        var filterData = self.getFilterObjAndByVal();
-                        $("#current-filter").show().find(".text").text(filterData.bookmarkText);
-                        $("#connector-container").show();
-                    }, 500);
-                }
-            }, 500);
+    computed: {
+        slippingAwayUsers: function() {
+            return this.$store.state.countlySlippingAwayUsers.slippingAwayUsers;
+        },
+        isLoading: function() {
+            return this.$store.state.countlySlippingAwayUsers.isLoading;
         }
     },
-    loadAndRefresh: function() {
-        var filter = {};
-        for (var i in this.filterObj) {
-            filter[i.replace("up.", "")] = this.filterObj[i];
+    methods: {
+        onUserListClick: function(timeStamp) {
+            var data = {
+                "lac": {"$lt": timeStamp}
+            };
+            var currentFilters = this.$store.state.countlySlippingAwayUsers.slippingAwayUsersFilters;
+            if (currentFilters.query) {
+                Object.assign(data, countlySlippingAwayUsers.helpers.buildFilters(currentFilters));
+            }
+            window.location.hash = '/users/query/' + JSON.stringify(data);
         }
-        this._query = filter;
-        app.navigate("/analytics/slipping-away/" + JSON.stringify(filter), false);
-        this.refresh();
+    }
+});
+
+var SlippingAwayUsersPage = countlyVue.components.BaseComponent.extend({
```

Comment:
By convention view names should be suffixed with "View". So it should be SlippingAwayUsersView. In addition, views should extend countlyVue.views.BaseView. This would save you from importing i18n and a few other mixins.